src/types/models.ts

```diff
@@ -0,0 +1,20 @@
+export interface User {
+  id: number;
+  name: string;
+}
+
+export interface Commit {
+  id: string;
+  user_id: number;
+  message: string;
+  committed_at: Date;
+}
+
+export interface CommitDiagnosis {
+  id: number;
+  commit_id: string;
+  rule: string;
+  data?: {
```

Comment:
This should be a generic JSON object: `Record<string, unknown>` should be sufficient.